sdcm/sct_events.py

```diff
@@ -23,7 +23,7 @@
 LOGGER = logging.getLogger(__name__)
 
 
-manager = Manager()
+MANAGER = Manager()
```

Comment:
where is it used?

Reply:
it's not... weird pylint didn't yell at it

Reply:
removed